Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -124,7 +125,7 @@ + (NSString*) userAgentString {
             BNCPreferenceHelper *preferences = [BNCPreferenceHelper preferenceHelper];
             preferences.browserUserAgentString = browserUserAgentString;
             preferences.lastSystemBuildVersion = self.systemBuildVersion;
-			//NSLog(@"[Branch] userAgentString: '%@'.", browserUserAgentString);
+			BNCLogDebug(@"userAgentString: '%@'.", browserUserAgentString);
```

Comment:
Ok. Thanks for clarifying. What about the `suppressWarningLogs`? AFAICT, it is not checked when logging warnings. Am I missing something?